lib/aws.js

```diff
@@ -33,8 +33,7 @@ AWS.util.update(AWS.Config.prototype.keys, {
     var credentials = null;
     new AWS.CredentialProviderChain([
       function () { return new AWS.EnvironmentCredentials('AWS'); },
-      function () { return new AWS.EnvironmentCredentials('AMAZON'); },
-      function () { return new AWS.SharedIniFileCredentials(); }
```

Comment:
I don't think we can safely remove the SharedIniFileCredentials from being invoked when the SDK is loaded.

While it's true that the defaultProviders will be called sequentially when a request is made and no valid/current credentials were loaded, there may be users that are performing logic on the credentials prior to making any requests. Removing this has the potential to break them.